File: src/test/java/com/koddy/server/common/mock/stub/StubEmailSender.java
package com.koddy.server.common.mock.stub;

import com.koddy.server.mail.application.adapter.EmailSender;
import lombok.extern.slf4j.Slf4j;

@Slf4j
public class StubEmailSender implements EmailSender {
    @Override
    public void sendEmailAuthMail(final String targetEmail, final String authCode) {
        log.info("학교 인증 메일 발송 -> Email=[{}], AuthCode=[{}]", targetEmail, authCode);
    }
}

File: src/test/java/com/koddy/server/common/mock/stub/StubEmailSender.kt
package com.koddy.server.common.mock.stub

import com.koddy.server.global.log.logger
import com.koddy.server.mail.application.adapter.EmailSender
import org.slf4j.Logger

open class StubEmailSender : EmailSender {
    private val log: Logger = logger()

    override fun sendEmailAuthMail(
        targetEmail: String,
        authCode: String,
    ) = log.info("학교 인증 메일 발송 -> Email=[{}], AuthCode=[{}]", targetEmail, authCode)
}
